ui/src/main/java/com/epam/eco/kafkamanager/ui/config/KafkaManagerUiProperties.java

```diff
@@ -24,7 +26,10 @@
 public class KafkaManagerUiProperties {
 
     private String gaTrackingId;
+    private Map<String, String> dataCatalogTool;
```

Comment:
Please use pojo instead of map e.g.:
```
private UrlTemplate dataCatalogTool;
...
class UrlTemplate {
private String name;
private String url;
...
}
```